cdap-docs/vars

```diff
@@ -0,0 +1,11 @@
+BUILD_TEMP="build-temp"
+COMMON="_common"
+COMMON_HTACCESS="$COMMON/htaccess"
+COMMON_IMAGES="$COMMON/_images"
+COMMON_SOURCE="$COMMON/_source"
+COMMON_CONF_PY="$COMMON/common_conf.py"
+COMMON_HIGHLEVEL_PY="$COMMON/highlevel_conf.py"
+COMMON_PLACEHOLDER="$COMMON/_source/placeholder_index._rst"
+HTACCESS="htaccess"
+
+MANUALS="admin-manual developers-manual examples-manual integrations reference-manual"
```

Comment:
I am thinking there should be one source from various scripts that need it.  common_conf.py is a script, so it makes less sense to me to have it there (a bash script sourcing a python script for variables seems odd to me).